src/aks-preview/HISTORY.rst

```diff
@@ -16,10 +16,6 @@ Pending
 +++++++
 * Add `--disable-workload-identity` to the `az aks update` command.
 
-0.5.156
```

Comment:
maybe you should keep this version history as it has been released

Reply:
Is there any updates for this comment?